corehq/apps/couch_sql_migration/tests/test_migration.py

```diff
@@ -146,8 +170,16 @@ def test_submission_error_log_migration(self):
         self._compare_diffs([])
 
     def test_hqsubmission_migration(self):
```

Comment:
I guess the name makes sense and it's not a crazy idea (guessing we figured we would split them out in reports and stuff). too bad we were never consistent with it.